Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'turbolinks' # improves speed of following links in web application
 gem 'uglifier' # compressor for JavaScript assets
 
 # Stanford gems
-gem 'assembly-image'
+gem 'assembly-image', '~> 1.7.1'
```

Comment:
Why are you pinning so pessimistically?  Would `~> 1.7` suffice?

Reply:
Fixed. More optimistic.